### PR TITLE
chore(aws-events): rebrand of CloudWatch Events to EventBridge

### LINKED ISSUE #7088: CloudWatch Events or EventBridge?
As per the comment I raised in https://github.com/aws/aws-cdk/issues/4455#issuecomment-606473416 it's kind of confusing that all of the doco talks about `CloudWatch Events`, yet the service is called `EventBridge`. Did this get rebranded or something? Should the doco be updated to better match the current reality?

- https://docs.aws.amazon.com/cdk/api/latest/docs/aws-events-readme.html
  - > Amazon **CloudWatch Events** Construct Library
- https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/AWS_Events.html
  - > Amazon **EventBridge** Resource Type Reference

---

This is a 📕 documentation issue

### PR DESCRIPTION
**Pull request by SomayaB**
### Commit Message
chore(aws-events): rebrand of CloudWatch Events to EventBridge 

Cloudwatch Events is getting rebranded to EventBridge so we're updating the terminology and links in the aws-events and aws-events-targets modules

Closes #7088
### End Commit Message

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Comment by Jerry-AWS** [comment]
I've received guidance on the EventBridge branding in our dev guides and have updated the CDK Dev Guide accordingly. (The only mention of CloudWatch Events was actually a reference to events in a more general sense, so I've just reworded to make that clear.) So this PR can proceed.

File: packages/@aws-cdk/aws-events-targets/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Event Targets for AWS CloudWatch Events
+# Event Targets for AWS EventBridge
```
Comment (shivlaks) [inline packages/@aws-cdk/aws-events-targets/README.md:1]:
```suggestion
# Event Targets for Amazon EventBridge
```

File: packages/@aws-cdk/aws-events-targets/README.md [hunk 2]
```diff
@@ -7,7 +7,7 @@
 ---
 <!--END STABILITY BANNER-->
 
-This library contains integration classes to send AWS CloudWatch Events to any
+This library contains integration classes to send AWS EventBridge to any
```
Comment (shivlaks) [inline packages/@aws-cdk/aws-events-targets/README.md:10]:
```suggestion
This library contains integration classes to send Amazon EventBridge to any
```

File: packages/@aws-cdk/aws-events-targets/lib/codebuild.ts [hunk 3]
```diff
@@ -12,13 +12,13 @@ export interface CodeBuildProjectProps {
...
    *
-   * @default - the entire CloudWatch event
+   * @default - the entire EventBridge event
    */
   readonly event?: events.RuleTargetInput;
 }
 
 /**
- * Start a CodeBuild build when an AWS CloudWatch events rule is triggered.
+ * Start a CodeBuild build when an AWS EventBridge rule is triggered.
```
Comment (shivlaks) [inline packages/@aws-cdk/aws-events-targets/lib/codebuild.ts:21]:
```suggestion
 * Start a CodeBuild build when an Amazon EventBridge rule is triggered.
```

File: packages/@aws-cdk/aws-events-targets/lib/sns.ts [hunk 4]
```diff
@@ -9,13 +9,13 @@ export interface SnsTopicProps {
...
    *
-   * @default the entire CloudWatch event
+   * @default the entire EventBridge event
    */
   readonly message?: events.RuleTargetInput;
 }
 
 /**
- * Use an SNS topic as a target for AWS CloudWatch event rules.
+ * Use an SNS topic as a target for AWS EventBridge rules.
```
Comment (shivlaks) [inline packages/@aws-cdk/aws-events-targets/lib/sns.ts:18]:
```suggestion
 * Use an SNS topic as a target for Amazon EventBridge rules.
```

File: packages/@aws-cdk/aws-events-targets/lib/sqs.ts [hunk 5]
```diff
@@ -21,14 +21,14 @@ export interface SqsQueueProps {
...
-   * @default the entire CloudWatch event
+   * @default the entire EventBridge event
    */
   readonly message?: events.RuleTargetInput;
 
 }
 
 /**
- * Use an SQS Queue as a target for AWS CloudWatch event rules.
+ * Use an SQS Queue as a target for AWS EventBridge rules.
```
Comment (shivlaks) [inline packages/@aws-cdk/aws-events-targets/lib/sqs.ts:31]:
```suggestion
 * Use an SQS Queue as a target for Amazon EventBridge rules.
```

File: packages/@aws-cdk/aws-events-targets/lib/state-machine.ts [hunk 6]
```diff
@@ -10,13 +10,13 @@ export interface SfnStateMachineProps {
...
    *
-   * @default the entire CloudWatch event
+   * @default the entire EventBridge event
    */
   readonly input?: events.RuleTargetInput;
 }
 
 /**
- * Use a StepFunctions state machine as a target for AWS CloudWatch event rules.
+ * Use a StepFunctions state machine as a target for AWS EventBridge rules.
```
Comment (shivlaks) [inline packages/@aws-cdk/aws-events-targets/lib/state-machine.ts:19]:
```suggestion
 * Use a StepFunctions state machine as a target for Amazon EventBridge rules.
```

File: packages/@aws-cdk/aws-events-targets/package.json [hunk 7]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@aws-cdk/aws-events-targets",
   "version": "0.0.0",
-  "description": "Event targets for AWS CloudWatch Events",
+  "description": "Event targets for AWS EventBridge",
```
Comment (shivlaks) [inline packages/@aws-cdk/aws-events-targets/package.json:4]:
```suggestion
  "description": "Event targets for Amazon EventBridge",
```

File: packages/@aws-cdk/aws-events/package.json [hunk 8]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@aws-cdk/aws-events",
   "version": "0.0.0",
-  "description": "AWS CloudWatch Events Construct Library",
+  "description": "AWS EventBridge Construct Library",
```
Comment (shivlaks) [inline packages/@aws-cdk/aws-events/package.json:4]:
```suggestion
  "description": "Amazon EventBridge Construct Library",
```

**Comment by rix0rrr** [comment]
Thanks Somaya! 

For some reason, our linter is now complaining:

```
@aws-cdk/aws-events: - [package-info/README.md] The title of the README.md file must be "Amazon CloudWatch Events Construct Library" (fixable)
```

Looks like you need to update the `package.json` as well.

**Comment by SomayaB** [comment]
> Thanks Somaya!
> 
> For some reason, our linter is now complaining:
> 
> ```
> @aws-cdk/aws-events: - [package-info/README.md] The title of the README.md file must be "Amazon CloudWatch Events Construct Library" (fixable)
> ```
> 
> Looks like you need to update the `package.json` as well.

@rix0rrr It looks like it's coming from [aws-service-official-names.json](https://github.com/aws/aws-cdk/blob/master/tools/pkglint/lib/aws-service-official-names.json) and not the package.json, is that right?

File: packages/@aws-cdk/aws-events-targets/README.md [hunk 9]
```diff
@@ -7,7 +7,7 @@
 ---
 <!--END STABILITY BANNER-->
 
-This library contains integration classes to send AWS CloudWatch Events to any
+This library contains integration classes to send Amazon EventBridge to any
```
Comment (0xdevalias) [inline packages/@aws-cdk/aws-events-targets/README.md:10]: Should be ‘Amazon EventBridge Events’?

**Pull request merged**